pkg/k8shandler/certificates.go

```diff
@@ -68,7 +68,7 @@ func (clusterRequest *ClusterLoggingRequest) extractSecretToFile(secretName stri
 
 	// check to see if the map value exists
 	if !ok {
-		logger.Infof("No secret data %q found", key)
+		logger.Warnf("No secret data %q found. Please be aware but likely not an issue for deprecated keys: app-ca,app-key,app-cert,infra-ca,infra-key,infra-cert", key)
```

Comment:
then why not make the message `Info` level for deprecated keys, but `Warn` level for everything else?

Reply:
better?